libs/core/src/lib/wizard/wizard-next-step/wizard-next-step.component.ts

```diff
@@ -11,4 +11,14 @@ export class WizardNextStepComponent {
      */
     @Input()
     floating = false;
+
+    /** @hidden */
+    displayed = true;
+
+    constructor(private _elRef: ElementRef) {}
+
+    /** @hidden */
+    getElRef(): ElementRef {
```

Comment:
According to the new agreement this should be named `_getElRef()` since it's public but hidden